ext/parquet/decoder_test.go

```diff
@@ -42,7 +41,7 @@ var _ = Describe("Decoder", func() {
 			Bool:       true,
 			Float:      f32ptr(0),
 			DateString: "03/01/09", ByteString: []byte("0"),
-			Timestamp: time.Unix(1235865600, 0),
+			// Timestamp: time.Unix(1235865600, 0),
```

Comment:
This failed to decode. Checking.

Reply:
Pls

Reply:
?

Reply:
I'm not sure what the type it that has been encoded. It says ` found [12]uint8 instead` which looking at the support types is the int96. We've had issues with this in the past, i believe it's a deprecated column.

Reply:
Int96 is the right type, which is encoded as [12] byte

Reply:
we must provide support for timestamps, this change should be a drop in replacement, everything that we had before must still work. you can't just comment out tests

Reply:
the lib provides some utility functions https://github.com/fraugster/parquet-go/blob/master/int96_time.go#L29

Reply:
So reader resolves the schema to be 

```
message schema {
  optional int32 id;
  optional boolean bool_col;
  optional int32 tinyint_col;
  optional int32 smallint_col;
  optional int32 int_col;
  optional int64 bigint_col;
  optional float float_col;
  optional double double_col;
  optional binary date_string_col;
  optional binary string_col;
  optional int96 timestamp_col;
}
```

The only way I can see to resolve int96 to a time is to add a custom Unmarshal. I've looked at the way they fill the values of a struct from the schema and it's it won't map int96 to a ts by default. AFAIK.